README.md

```diff
@@ -172,8 +172,9 @@ Compatibility with Linux distributions based on Debian, Ubuntu or Redhat familie
 
 The continuous integration setup of this project currently covers following platforms:
 - CentOS 6, 7 and 8
-- Debian 8, 9 and 10
-- Ubuntu 14.04, 16.04, 18.04 and 20.04
+- Rocky Linux 8
+- Debian 8, 9, 10 and 11
+- Ubuntu 14.04, 16.04, 18.04, 20.04 and 21.10
```

Comment:
For other readers: Ubuntu 14/16 LTS lives have been extended, more information at https://ubuntu.com/blog/ubuntu-14-04-and-16-04-lifecycle-extended-to-ten-years